src/components/VueScriptComponent.vue

```diff
@@ -1,44 +1,48 @@
 <template>
-  <div :id="compId">
-  </div>
+  <div :id="compId" :key="id"></div>
```

Comment:
what is the purpose of the :key here?

Reply:
My apologies. It was a duplicate on my end.